app/models/consultation.rb

```diff
@@ -259,15 +266,22 @@ def neighbour_letter_body
       eaves_height: planning_application&.proposal_measurement&.eaves_height,
       current_user: Current.user.name,
       council_address: I18n.t("council_addresses.#{local_authority.subdomain}"),
-      application_link:)
+      application_link:
+    }
+
+    replace_placeholders(body, defaults)
   end
 
   def neighbour_letter_content
     "# #{neighbour_letter_header}\n\n#{neighbour_letter_body}"
   end
 
   def neighbour_letter_text
-    super.presence || neighbour_letter_content
+    if super.presence&.include?("{{")
+      neighbour_letter_content
+    else
+      super.presence
+    end
```

Comment:
Obviously not very happy with this. Can someone suggest a better way?

Reply:
I wonder if it's worth just unconditionally using `neighbour_letter_content`? Is there any downside to running the `replace_placeholders` part if there are no placeholders in it? I assume there will be _some_ performance impact, but probably tiny, relatively speaking.

Reply:
The problem with that is it then overwrites any edits that the officer makes in the text box